pkg/cloudprovider/gce/gce.go

```diff
@@ -225,6 +225,19 @@ func makeHostLink(projectID, zone, host string) string {
 		projectID, zone, host)
 }
 
+func makeComparableHostLink(zone, host string) string {
```

Comment:
These two functions have teh same name, but the words are in different order.  Can we do better?